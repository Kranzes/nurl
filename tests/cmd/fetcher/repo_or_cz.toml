args = ["https://repo.or.cz/tinycc.git", "release_0_9_27"]
