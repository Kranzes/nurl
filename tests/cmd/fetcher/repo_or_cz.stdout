fetchFromRepoOrCz {
  repo = "tinycc";
  rev = "release_0_9_27";
  hash = "sha256-5yRaB/gxk1Gm/rukAe0pm1PEl5NdieVFngp87jENtYo=";
}